L2 nightly moreh_linear test is failing

https://github.com/tenstorrent/tt-metal/actions/runs/27937785678/job/82665462891
```
FAILED tests/ttnn/nightly/unit_tests/operations/moreh/test_moreh_linear.py::test_moreh_bias_backward_fp32[shapes=([8, 512, 768], [768, 768], [1, 768], [8, 512, 768])] - RuntimeError: TT_FATAL @ /project/tt_metal/impl/tensor/tensor_apis.cpp:1058: tensor.padded_shape()[-2] % constants::TILE_HEIGHT == 0 && tensor.padded_shape()[-1] % constants::TILE_WIDTH == 0
info:
Last 2 dims of input shape must be multiples of 32
```

--- a/tests/ttnn/nightly/unit_tests/operations/moreh/test_moreh_linear.py
+++ b/tests/ttnn/nightly/unit_tests/operations/moreh/test_moreh_linear.py
@@ -478,8 +478,8 @@ def test_moreh_bias_backward_fp32(shapes, device):
     torch_output.backward(torch_output_grad.float())
     ## test for equivalance
     rtol = atol = 0.1
-    tt_bias_grad_fp32_cpu = tt_bias_grad_fp32.cpu().to(ttnn.ROW_MAJOR_LAYOUT).unpad_from_tile(bias_shape).to_torch()
-    tt_bias_grad_cpu = tt_bias_grad.cpu().to(ttnn.ROW_MAJOR_LAYOUT).unpad_from_tile(bias_shape).to_torch()
+    tt_bias_grad_fp32_cpu = ttnn.to_torch(tt_bias_grad_fp32)
+    tt_bias_grad_cpu = ttnn.to_torch(tt_bias_grad)
     passing, output_pcc = comp_allclose_and_pcc(
         torch_bias_fp32.grad, tt_bias_grad_fp32_cpu, pcc=0.98, rtol=rtol, atol=atol
     )

